apps/financial-aid/web-osk/src/components/Summary/AllFiles.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useContext } from 'react'
+import { Box, Button, Icon, Text } from '@island.is/island-ui/core'
+import { useRouter } from 'next/router'
+
+import * as styles from './summary.treat'
+
+import { FormContext } from '@island.is/financial-aid-web/osk/src/components/FormProvider/FormProvider'
+import { Routes } from '@island.is/financial-aid/shared/lib'
+
+const AllFiles = () => {
+  const router = useRouter()
+
+  const { form } = useContext(FormContext)
+
+  const allFiles = form.incomeFiles
+    .concat(form.taxReturnFiles)
+    .concat(form.otherFiles)
+
+  return (
+    <Box
+      display="flex"
+      justifyContent="spaceBetween"
+      alignItems="flexStart"
+      paddingY={[4, 4, 5]}
+      marginBottom={[2, 2, 5]}
+    >
+      <Box marginRight={3}>
+        <Text fontWeight="semiBold">Gögn</Text>
+        <Box>
+          {allFiles.map((file, index) => {
+            return (
+              <>
+                {file && (
```

Comment:
Is file at any point undefined? 
Think we can skip this check and the fragment on the line above